packages/ag-charts-website/src/content/docs/error-bars-test/_examples/temperatures/main.ts

```diff
@@ -1,6 +1,23 @@
-import { AgChartOptions, AgEnterpriseCharts } from 'ag-charts-enterprise';
+import { AgChartOptions, AgEnterpriseCharts, AgErrorBarFormatterParams } from 'ag-charts-enterprise';
 import { getData } from './data';
 
+
```

Comment:
Nit: Remove the excess lines here and line 18?

Reply:
done